rust_snuba/Cargo.lock

```diff
@@ -2191,9 +2191,9 @@ checksum = "0ab1bc2a289d34bd04a330323ac98a1b4bc82c9d9fcb1e66b63caa84da26b575"
 
 [[package]]
 name = "openssl"
-version = "0.10.64"
+version = "0.10.68"
```

Comment:
This version is 0.10.68 while the commit message says 0.10.64. Is this intended?

Reply:
Fixed